hack/update-helm.sh

```diff
@@ -92,3 +95,26 @@ for output_file in ${DEST_RBAC_DIR}/*.yaml; do
     $YQ -N -i '.subjects.[].namespace = "{{ .Release.Namespace }}"' $output_file
   fi
 done
+
+# Add visibility files, replace names, namespaces in helm format
+for output_file in ${DEST_VISIBILITY_DIR}/*.yaml; do
+  # The name of the v1alpha1.pending-workloads.kueue.x-k8s.io APIService needs to remain unchanged.
```

Comment:
make sure the comment is up-to-date, should this be `v1alpha1.visibility.kueue.x-k8s.io` now?

Reply:
Yes it should, thanks for pointing that out